web/static/css/artifact_page.css

```diff
@@ -71,10 +71,35 @@
   padding-top: 2rem;
   padding-bottom: 0.625rem;
 }
+#contents {
+  display: flex;
+  max-width: 1800px;
+  margin: auto;
+  gap: 3rem;
+  justify-content: center;
+}
+#contents .sidebar {
```

Comment:
Looks like the auto-scrolling height is not right. If I click the `explore` button, the root node is partially hidden. Should we give same top padding to the side bar?
<img width="1215" alt="image" src="https://github.com/VasuDevrani/artifact-explorer-oras/assets/4254263/84fad6b8-3038-47f7-bb81-3b5a667576f3">